### PR TITLE
Fix issue about loading configs from tsconfig.json

### PR DESCRIPTION
**Pull request by ashmna**

### CODE REVIEW

**Review by devongovett — APPROVED** [review]

File: packages/transformers/typescript-types/src/TSTypesTransformer.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {shake} from './shake';
 
 export default (new Transformer({
   async loadConfig({config, options}) {
-    await loadTSConfig(config, options);
+    return await loadTSConfig(config, options);
```
Comment (devongovett) [inline packages/transformers/typescript-types/src/TSTypesTransformer.js:19]: fixes lint

```suggestion
    return loadTSConfig(config, options);
```

File: packages/transformers/typescript-types/src/TSTypesTransformer.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {shake} from './shake';
 
 export default (new Transformer({
   async loadConfig({config, options}) {
```
Comment (devongovett) [inline packages/transformers/typescript-types/src/TSTypesTransformer.js]:
```suggestion
  loadConfig({config, options}) {
```

File: packages/transformers/typescript-types/src/TSTypesTransformer.js (hunk 1 above)
Comment (devongovett) [inline packages/transformers/typescript-types/src/TSTypesTransformer.js]: fixes lint

```suggestion
    return loadTSConfig(config, options);
```

**Review by findmet — APPROVED** [review]
Thanks for your help

**Review by findmet — APPROVED** [review]

**Pull request merged**